src/main/java/org/matsim/episim/EpisimPerson.java

```diff
@@ -214,9 +214,9 @@ public final class EpisimPerson implements Attributable {
 	private double antibodyLevelAtInfection = 0;
 
 	/**
-	 * Whether agent is part of subpopulation which is immune to infection
+	 * Immune response multiplier, which is used to scale the antibody increase due to an immunity event
 	 */
-	private ImmuneResponse immuneResponse = ImmuneResponse.normal;
+	private double immuneResponseMultiplier = 1.0;
```

Comment:
Ist das nicht teil des Snapshots? Wenn ja, wieso nicht?

Reply:
Wieso wird das nicht in "write" und "read" in den Snapshot geschrieben ? Der Wert würde verloren gehen, wenn vom Snapshot geladen wird.

Reply:
👍 good point, I've implemented the change